perfscale_regression_ci/scripts/common.sh

```diff
@@ -20,6 +20,26 @@ function wait_for_completion() {
   done
 }
 
+# pass $name_identifier $number
+# e.g. wait_for_running "job-" 100
+function wait_for_running() {
+  name_identifier=$1
+  number=$2
+  COUNTER=0
+  running=$(oc get pods -A | grep $name_identifier | grep -c Running)
+  while [ $running -lt $number ]; do
+    sleep 3
+    running=$(oc get pods -A | grep $name_identifier | grep -c Running)
+    echo "$running pods are running"
+    COUNTER=$((COUNTER + 1))
+    if [ $COUNTER -ge 1200 ]; then
+      not_running=$(oc get pods -A | grep $name_identifier | grep -v -c Running)
+      echo "$not_running pods are still not running after 20 minutes"
```

Comment:
If we reach this, we took an hour right? Not 20 min?

Reply:
took this out because I didn't need it